db/repos/transactions/index.js

```diff
@@ -334,7 +334,9 @@ const Queries = {
 				? ` AND ${params.owner}`
 				: params.owner,
 			params.sortField
-				? `ORDER BY ${[params.sortField, params.sortMethod].join(' ')}`
+				? `ORDER BY ${[params.sortField, params.sortMethod].join(
+						' '
+					)}, "t_rowId" ASC`
```

Comment:
@Tschakki Just thought of a scenario, what if while calling `db.transactions.list()` someone don't pass the `params.sortField` then this query will be invalid. To make sure this always works, we have to set a default value for `params.sortField` and `params.sortField` in `db.transactions.list()`

Reply:
Discard this comment, as I noticed that `sortField` check is already in place. The indentation of `?` make me think there is not.